p2p/net/swarm/swarm.go

```diff
@@ -260,10 +270,16 @@ type ps2netNotifee struct {
 
 func (n *ps2netNotifee) Connected(c *ps.Conn) {
 	n.not.Connected(n.net, inet.Conn((*Conn)(c)))
+	if metric, err := peersTotalGauge(n.net.LocalPeer()); err == nil {
+		metric.Inc()
+	}
 }
 
 func (n *ps2netNotifee) Disconnected(c *ps.Conn) {
 	n.not.Disconnected(n.net, inet.Conn((*Conn)(c)))
+	if metric, err := peersTotalGauge(n.net.LocalPeer()); err == nil {
```

Comment:
yeah, we shouldnt ship things with inconsistent labels. im on the fence (a debug log may be ok), but dont care much either way. i think @whyrusleeping would say panic.